LaunchScreenDemo/WebServices/Alamofire/Controller/RegisterUserViewController.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  RegisterUserViewController.swift
+//  LaunchScreenDemo
+//
+//  Created by Vidhi Patel on 04/07/23.
+//
+
+import UIKit
+
+class RegisterUserViewController: UIViewController {
+    
+    //MARK: - IBOutlets and Let
+    @IBOutlet private weak var registerBtn: UIButton!
+    @IBOutlet private weak var passwordTf: UITextField!
+    @IBOutlet private weak var emailTf: UITextField!
+    @IBOutlet private weak var usernameTf: UITextField!
+    @IBOutlet private weak var cardView: UIView!
+    private let viewModel: RegisterViewModel = RegisterViewModel()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        cardView.layer.cornerRadius = 20
+        bindViewModel()
+        bindSignUpUi()
+    }
+    
+    // MARK: - Private methods
+    private func bindViewModel() {
+        viewModel.validationError.bind { [weak self]
+            message in
+            self?.showAlertDialog(message: message, type: .error)
+        }
+    }
+    
+    private func bindSignUpUi() {
+        viewModel.error.bind { [weak self]
+            message in
+            print("Error \(message)")
+            self?.showAlertDialog(message: "Server side Error", type: TypeDialog.error)
+        }
+        
+        viewModel.signUpSuccess.bind { [weak self]
+            message in
+            print(message)
```

Comment:
remove unnecessary print statement as you are already showing message body on alert